app/controllers/victimCompensation.js

```diff
@@ -1,3 +1,8 @@
 var args = arguments[0] || {};
+var providerSearch = require('providerSearch');
 
-Alloy.Globals.addActionBarButtons($.win);
\ No newline at end of file
+if(Alloy.Globals.isAndroid){
+	providerSearch.createAndroidSearchBar($.win, $.providerList);
+}
+
+$.providerButtonBar.setProviderListObject($.providerList);
```

Comment:
I see a lot of warnings for no newline at end of file. Should be fixed for good style according to this dude:
http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file